### PR TITLE
Build: build a documentation site for the Playground

### PR DESCRIPTION
**Pull request by michaelpj**
This is a self-contained site that can be deployed with the
Playground.

We need to set the relative URLs that correspond to how things are going
to be served, but this can adjusted when we actually start doing that.

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -184,6 +184,16 @@ let
     };
 
     plutus-playground = rec {
+      documentation-site = let
+        # TODO: the playgroundUrl needs to be set to whatever will actually be appropriate when it's bundled
+        # with the playground
+        adjustedTutorial = docs.plutus-tutorial.override { playgroundUrl = "../.."; haddockUrl = "../haddock"; };
```
Comment (shmish111) [inline default.nix]: since this is being deployed to the same domain, can't you just use relative paths?

Comment (michaelpj) [reply default.nix]: Those are relative paths, no? I just mean we'll need to tweak it for whatever the exact layout.

Comment (shmish111) [reply default.nix]: I mean setting the `playgroundUrl`, why would we do that? Can you give me an example.

Comment (michaelpj) [reply default.nix]: The `playgroundUrl` is for the docs to link to the playground. But we could put the playground in a number of places, e.g. we could put it at the root or at `/playground`, say.

It's natural to make it configurable from the documentation's point of view, so we just have to say what it actually is here.

Comment (shmish111) [reply default.nix]: got it

**Pull request merged**